content/doc/tutorials/download-and-run-jenkins-in-docker.adoc

```diff
@@ -0,0 +1,137 @@
+=== Download and run Jenkins in Docker
+
+In this tutorial, we'll be running Jenkins as a Docker container from a Docker
+image of Jenkins with Blue Ocean (pre-bundled). Read more about these concepts
+in the Docker documentation's https://docs.docker.com/get-started/[Getting
+Started, Part 1: Orientation and setup] page and the
+https://docs.docker.com/engine/docker-overview/[Docker overview] page for an
+extensive overview of Docker's capabilities.
+
+TIP: If you've already run though this procedure from link:..[another
+introductory tutorial], then you can skip this section and
+<<fork-and-clone-the-sample-repository-on-github,jump to the next>>.
+
+. Open up a terminal/command prompt window.
+. Pull the https://hub.docker.com/r/jenkinsci/blueocean/[latest Docker image for
+Jenkins with Blue Ocean (pre-bundled)] by running the command: +
+  `docker pull jenkinsci/blueocean` +
+  This command downloads the latest Docker image for Jenkins with Blue Ocean
+  that will run Jenkins as a container in your locally running Docker instance.
+. Create the following Docker data volumes that will be used to persistently
+store all data in the Jenkins home directory and Maven repository (explained in
+more detail later) by running the commands: +
+  `docker volume create jenkins-data` +
+  and +
+  `docker volume create maven-repository`
++
+--
+NOTE: These data volumes allow you to shut down Docker (for example, part way
+through the tutorial) without you losing any work you've already done.
+--
++
+. Start the downloaded Jenkins/Blue Ocean Docker image (for running this
+tutorial) using the appropriate command:
+* *On Mac OS X:*
++
+----
+docker run --rm \
+  --name jenkins-tutorials \
+  -u root \
+  -p 8080:8080 \
+  -v jenkins-data:/var/jenkins_home \
+  -v maven-repository:/root/.m2 \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  -v /Users:/Users \
+  jenkinsci/blueocean
+----
++
+* *On Linux:* +
+  Use the Mac OS X command above but replace `-v /Users:/Users` with
+  `-v /home:/home`
```

Comment:
Maybe we need to write the docker command out in full for Linux users - I feel we're treating them badly by omitting this.

Reply:
I agree!  At the very least, I would not say, "Use the Mac OS X command above..."; say something like "Use the command shown for Mac OS X above..." or something

Reply:
@gilesgas - What are the paths in the docker container instance that are actually being mapped?  There's probably environment variables on in the docker instance and on the host that will unify this.

Reply:
@StackScribe , thanks for confirming my hunch ... Also, following @bitwiseman 's suggestion and subsequent discussion with him, I've ended up combining the Mac OS X and Linux commands together, since I'm now recommending the option '-v "$HOME":"$HOME"' for both Mac OS X and Linux (which makes both their 'docker run ...' commands identical now). For Windows, I'm recommending that this option is '-v "%HOMEPATH%":"$HOME"', which I haven't tested yet, but from an educated guess, I'm assuming this will work. I've found very little literature on how to refer to Windows env variables in 'docker run ...' commands.